frontend/routes/ConfigRoute.tsx

```diff
@@ -0,0 +1,26 @@
+import * as React from 'react';
+import { BreadcrumbActions, GameVisibilityActions } from '../storage/common';
+import { connect } from 'react-redux';
+import { Dispatch, AnyAction } from 'redux';
+import { ConfigView } from '../views/ConfigView';
+
+interface ConfigRouteProps {
+	dispatch: Dispatch<AnyAction>;
+}
+
+export class ConfigRoute extends React.Component<ConfigRouteProps> {
+	public componentDidMount() {
+		this.props.dispatch(BreadcrumbActions.setBreadcrumb('Config'));
+	}
+	public render() {
+		const buttonCallback = (allVisible: boolean) => {
```

Comment:
When renaming the prop in `ConfigView` remember to rename this instance here too.